### ISSUE TITLE
Python does not build

### ISSUE THREAD

**Issue by mato** [rumpkernel/rumprun-packages#37]
https://hub.docker.com/r/mato/rumprun-packages-hw-x86_64/builds/b5cpv4cntfz4buuj5f7u8ml/

```
PermissionError: [Errno 13] Permission denied: '/usr/lib/python3.4/site-packages/' 
make[1]: Leaving directory '/build/rumprun-packages/python/build' make: Leaving directory '/build/rumprun-packages/python' 
```

**Comment by anttikantee** [comment]
Expected.  See README.

**Comment by mato** [comment]
In that case (make is expected to fail), I'll have to keep it out of the rumprun-packages autobuild.

Alternatively, it is not "forbidden" to patch build systems in packages (see MySQL), could you make it work without failing in one command?

**Comment by anttikantee** [comment]
ping @rday 
Can you fix the Makefile to be compatible with automated build testing?

**Comment by mato** [comment]
Still failing, different errors now:
https://hub.docker.com/r/mato/rumprun-packages-hw-x86_64/builds/bxzdh5sk7pcgutlh6ridwjk/

**Comment by rday** [comment]
Fixed the Makefile error in eddb5cadc29762f2c3278cf5587ef0525613128f. I tested this out using the Docker process you've setup as well. Let me know if you still hit an issue.
